src/resources/Notifications/notifications.ts

```diff
@@ -0,0 +1,13 @@
+import API from '../../APICore';
+import Resource from '../Resource';
+import Subscriptions from './Subscriptions/Subscriptions';
+
+export default class Notifications extends Resource {
```

Comment:
I think we should use the name from Swagger to reflect the API. remove s

Reply:
:+1: